src/vuex/modules/connection.js

```diff
@@ -62,6 +66,14 @@ export default function({ apollo }) {
     async persistNetwork(store, network) {
       localStorage.setItem(`network`, JSON.stringify(network.id))
     },
+    async preloadNetworkCapabilities({ commit }) {
+      const { data } = await apollo.query({
+        query: NetworksAll,
+        variables: { experimental: config.development },
```

Comment:
no... session.experimentalMode

Reply:
which comes from there (line 9 in `session.js`) 🤷‍♀

Reply:
Nonono! It should load all networks. And should be filtered where it's needed.

Reply:
Maybe we shouldn't filter networks here. I think that we have to load all possible network and then do something with enabled/experimental in components as now we have network-related links

Reply:
in which situation would a user see a link to an experimental network if he didn't before load experimental networks?

Reply:
If someone will send him a link

Reply:
this makes sense

Reply:
Maybe you should open an issue about this @iambeone